solidity/random-beacon/tasks/staking.ts

```diff
@@ -0,0 +1,110 @@
+import { task, types } from "hardhat/config"
+
+import type { HardhatRuntimeEnvironment } from "hardhat/types"
+
+task(
+  "staking",
+  "Stakes T tokens, increases authorization and registers the operator "
+)
+  .addOptionalParam("owner", "Staking owner address", undefined, types.string)
+  .addOptionalParam("provider", "Staking provider", undefined, types.string)
+  .addOptionalParam("operator", "Staking operator", undefined, types.string)
+  .addOptionalParam("amount", "Staking amount", 1_000_000, types.int)
+  .addOptionalParam(
+    "authorization",
+    "Authorization amount",
+    undefined,
```

Comment:
40_000 is the default minimum authorization for the beacon, and we should use it here.

```suggestion
    "authorizationAmount",
    "Authorization amount (default: minimumAuthorization)",
    undefined,
```